frontend/helpers/colorOverrides.ts

```diff
@@ -75,7 +75,7 @@ export const addColorOverrides = (overrides) => {
 
 				for (let j = 0; j < modelsList.length; j++) {
 					const { meshes, teamspace, modelId } = modelsList[j] as any;
-					Viewer.overrideMeshColor(teamspace, modelId, meshes, color);
+					await Viewer.overrideMeshColor(teamspace, modelId, meshes, color);
```

Comment:
This shouldn't need await, overrideMeshColor of different models shouldn't require one to be processed before another.

Not to mention the unity doesn't return a promise upon it's done.